app/viewmodels/SummaryCardHelper.scala

```diff
@@ -375,4 +378,23 @@ class SummaryCardHelper @Inject()(link: views.html.components.link) extends Impl
       }
     }
   }
+
+  def showDueOrPartiallyPaidDueTag(financial: Option[Financial])(implicit messages: Messages): Tag = if (financial.isDefined){
+      val amountDue = financial.get.amountDue
+      val outstandingAmountDue = financial.get.outstandingAmountDue
+      if (outstandingAmountDue < amountDue && outstandingAmountDue > 0) {
```

Comment:
In case of outstandingAmountDue=0.0 and amountDue = 0.0 we'd get a DUE tag, how are we handling this case.

Reply:
The function is only called in cases where something is due. Please correct me if I'm wrong, but in the case that the user has 0 outstanding amount and/or 0 amount due, wouldn't that mean they have fully paid their penalty and/or they don't have anything to pay. Which is covered by a different case in tagStatus()

Reply:
Discussed on dev hangout, and has been resolved 👍